instrumentation/opentelemetry-instrumentation-aiohttp-client/tests/test_aiohttp_client_integration.py

```diff
@@ -803,6 +805,24 @@ async def do_request(url):
         )
         self.memory_exporter.clear()
 
+    @mock.patch.dict(
+        os.environ, {"OTEL_PYTHON_AIOHTTP_CLIENT_EXCLUDED_URLS": "/some/path"}
```

Comment:
Could we also have the same tests added for regular `OTEL_PYTHON_EXCLUDED_URLS`?